<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>classEditWindow</class>
 <widget class="QMainWindow" name="classEditWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>374</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>374</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ویرایش دسته</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/edit.png</normaloff>../icons/edit.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="mainGroupBox">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>361</width>
      <height>266</height>
     </rect>
    </property>
    <property name="title">
     <string>ویرایش دسته</string>
    </property>
    <widget class="QGroupBox" name="settingGroupBox">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>341</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="classNameLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>نام دسته:</string>
      </property>
     </widget>
     <widget class="QLabel" name="folderPathLabel">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>40</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>آدرس پوشه:</string>
      </property>
     </widget>
     <widget class="QLabel" name="usenameLabel">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کاربری کانال:</string>
      </property>
     </widget>
     <widget class="QLabel" name="classPlaylistNameLabel">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>لیست پخش:</string>
      </property>
     </widget>
     <widget class="QLabel" name="classAparatCategoryLabel">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>100</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>دسته بندی آپارات:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="classNameLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="folderPathLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="classPlaylistNameLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="classAparatCategoryLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="passwordLabel">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>160</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>رمز عبور کانال:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="cancelPushButton">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>230</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>انصراف</string>
     </property>
    </widget>
    <widget class="QPushButton" name="savePushButton">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>230</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ذخیره</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
